platformtree/Cargo.toml

```diff
@@ -7,5 +7,8 @@ authors = ["Zinc Developers <zinc@github.com>"]
 name = "platformtree"
 plugin = true
 
+[dependencies]
```

Comment:
That might work for non-release builds, however when I run `cargo build --target=thumbv7em-none-eabi --release` in my downstream, I get `extern crate regex; can't find crate` so I'm inclined to leave it in `dependencies` so all builds work.